src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,16 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+  if (number % 5 == 0 && number % 3 == 0) {
+    return "FizzBuzz";
+  }
+
+  if (number % 5 == 0) {
+    return "Buzz";
+  }
+
+  if (number % 3 == 0) {
+    return "Fizz";
+  }
+  return number;
```

Comment:
should always use strict equality `===` instead of the loose equality operator `==` to avoid unexpected behaviour. [You can read more about the differences here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#:~:text=Finally%2C%20if%20both%20values%20are,0%20and%20one%20is%20%2D0%20.&text=Strict%20equality%20is%20almost%20always,is%20only%20equal%20to%20itself.)

Reply:
because you're checking the expressions `number % 5 === 0` and `number % 3 === 0` multiple times, you can assign these to a variable to make it cleaner and more readable. e.g.
```
const isDivisibleBy5 = number % 5 === 0;
const isDivisibleBy3 = number % 3 === 0;

if (isDivisibleBy5 && isDivisibleBy3) return "FizzBuzz";

if (isDivisibleBy5) return "Buzz";

if (isDivisibleBy3) return "Fizz";

return number;
```